python/prodV4/worker.py

```diff
@@ -1,10 +1,9 @@
 import os
+import pathlib
 import sys
 
-from pathlib import Path
```

Comment:
Any specific reason to remove it?

Reply:
Making it more consistent with the other imports in that file, and "import pathlib" and "pathlib.Path" was how the module was used in constants.py before